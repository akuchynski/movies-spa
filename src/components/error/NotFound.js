import React from "react";

export const NotFound = () => {
    return (
        <div className="content">
            <h2>Page not found!</h2>
        </div>
    );
};