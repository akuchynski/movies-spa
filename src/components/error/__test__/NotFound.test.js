import React from "react";
import { render } from "@testing-library/react";
import { NotFound } from "../NotFound";

describe('error components tests', () => {
    it('should render not found component', () => {
        const { asFragment } = render(<NotFound />);
        expect(asFragment()).toMatchSnapshot();
    });
});