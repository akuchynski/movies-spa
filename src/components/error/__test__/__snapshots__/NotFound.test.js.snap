// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error components tests should render not found component 1`] = `
<DocumentFragment>
  <div
    class="content"
  >
    <h2>
      Page not found!
    </h2>
  </div>
</DocumentFragment>
`;
